zerver/tests/test_saved_replies.py

```diff
@@ -0,0 +1,79 @@
+from zerver.lib.saved_replies import do_create_saved_reply, do_get_saved_replies
+from zerver.lib.test_classes import ZulipTestCase
+from zerver.models import SavedReply, UserProfile
+
+
+class SavedReplyTests(ZulipTestCase):
+    def get_user(self) -> UserProfile:
+        user = self.example_user("hamlet")
+
+        SavedReply.objects.filter(user_profile=user).delete()
+        return user
```

Comment:
I'm not totally sure why this helper would be needed; the test database won't come with any `SavedReply` objects unless we add them in `populate_db` ... which we might want to do, but only in the "dev environment only" section, not the section shared with the test suite.

And the database state gets reset at the end of each test using transactions.

Reply:
Removed.